torchaudio/prototype/models/hdemucs.py

```diff
@@ -929,3 +929,52 @@ def _ispectro(z: torch.Tensor, hop_length: int = 0, length: int = 0, pad: int =
     _, length = x.shape
     other.append(length)
     return x.view(other)
+
+
+def hdemucs_low(sources: List[str], sample_rate: int) -> HDemucs:
```

Comment:
remember to also declare these functions to `torchaudio/prototype/models/__init__.py`